### PR TITLE
fix: align names on monorepo packages

### PR DESCRIPTION
**Pull request by mrednic-1A**
## Related issues

- :bug: Fixes #(issue)
- :rocket: Feature #(issue)

### CODE REVIEW

**Review by matthieu-crouzet — DISMISSED** [review]

**Review by kpanot — DISMISSED** [review]

**Review by fpaul-1A — APPROVED** [review]

File: packages/@o3r/core/schematics/library/rules/rules.ng.ts [hunk 1]
```diff
@@ -40,7 +43,13 @@ export function ngGenerateModule(options: NgGenerateModuleSchema & { targetPath:
     return chain([
       mergeWith(templateNg, MergeStrategy.Overwrite),
       updatePackageDependenciesFactory(options.targetPath, otterVersion!, o3rCorePackageJson, options),
-      updateNgPackagrFactory(options.targetPath)
+      updateNgPackagrFactory(options.targetPath),
+      (t) => {
+        const packageJson = t.readJson(path.posix.join(options.targetPath, 'package.json')) as PackageJson;
```
Comment (kpanot) [inline packages/@o3r/core/schematics/library/rules/rules.ng.ts:48]: `path.posix.join(options.targetPath, 'package.json')` could set in `const`

File: packages/@o3r/core/schematics/sdk/rules/update-ts-paths.rule.ts [hunk 2]
```diff
@@ -30,10 +30,10 @@ export function updateTsConfig(targetPath: string, projectName: string, scope: s
 
     configWithPath.content.compilerOptions ||= {};
     configWithPath.content.compilerOptions.paths ||= {};
-    configWithPath.content.compilerOptions.paths[`@${scope}/${projectName}`] = [
+    configWithPath.content.compilerOptions.paths[`${scope ? '@' + scope + '/' : ''}${projectName}`] = [
```
Comment (kpanot) [inline packages/@o3r/core/schematics/sdk/rules/update-ts-paths.rule.ts:33]:
```suggestion
    configWithPath.content.compilerOptions.paths[`${scope ? `@${scope}/` : ''}${projectName}`] = [
```

File: packages/@o3r/core/schematics/sdk/rules/update-ts-paths.rule.ts [hunk 3]
```diff
@@ -30,10 +30,10 @@ export function updateTsConfig(targetPath: string, projectName: string, scope: s
...
       `${relativeTargetPath}/src/index`
     ];
-    configWithPath.content.compilerOptions.paths[`@${scope}/${projectName}/*`] = [
+    configWithPath.content.compilerOptions.paths[`${scope ? '@' + scope + '/' : ''}${projectName}/*`] = [
```
Comment (kpanot) [inline packages/@o3r/core/schematics/sdk/rules/update-ts-paths.rule.ts:36]:
```suggestion
    configWithPath.content.compilerOptions.paths[`${scope ? `@${scope}/`: ''}${projectName}/*`] = [
```

**Review by kpanot — APPROVED** [review]

File: packages/@o3r/core/schematics/library/rules/rules.ng.ts (hunk 1 above)
Comment (mrednic-1A) [reply packages/@o3r/core/schematics/library/rules/rules.ng.ts:48]: https://github.com/AmadeusITGroup/otter/pull/905

File: packages/@o3r/core/schematics/sdk/rules/update-ts-paths.rule.ts (hunk 2 above)
Comment (mrednic-1A) [reply packages/@o3r/core/schematics/sdk/rules/update-ts-paths.rule.ts:33]: done in https://github.com/AmadeusITGroup/otter/pull/905

File: packages/@o3r/core/schematics/sdk/rules/update-ts-paths.rule.ts (hunk 3 above)
Comment (mrednic-1A) [reply packages/@o3r/core/schematics/sdk/rules/update-ts-paths.rule.ts:36]: done in 
https://github.com/AmadeusITGroup/otter/pull/905

**Pull request merged**